rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/test_are_all_terminal_heating_loops_purchased_heating.py

```diff
@@ -46,14 +54,14 @@ def test__TEST_RMD__is_valid():
     ], f"Schema error: {schema_validation_result['error']}"
 
 
-def test__all_terminal_heating_loops_purchased_heating():
+def test__are_all_terminal_heating_loops_purchased_heating__all_terminal_heating_loops_purchased_heating():
     assert (
         are_all_terminal_heating_loops_purchased_heating(TEST_RMD, ["terminal_1"])
```

Comment:
Because we are going to need to fix a number of other unit tests due to conflating RMD and RMI, it might be good to start keeping them straight here.  `TEST_RMD` should really be `TEST_RMI`. `TEST_RMD_FULL` could just be `TEST_RMD`.

Reply:
Great point! Addressed.